documentation/docs/pages/docs/changelog.mdx

```diff
@@ -17,6 +17,13 @@
 
 all release branches are deployed through `release/VERSION_NUMBER` branches
 
+# 0.28.1-beta - 17th October 2025
+
+### Bug fixes
+-------------------------------------------------
+- fix: add tls for hosted clickhouse databases 
+- fix: ClickHouse now uses single database from `CLICKHOUSE_DB` env var instead of requiring `CREATE DATABASE` privileges, rationale: giving all permissions to one script to drop any database is unreasonable.
```

Comment:
> ClickHouse now uses single database from `CLICKHOUSE_DB` env var instead of requiring `CREATE DATABASE` privileges, rationale: giving all permissions to one script to drop any database is unreasonable.

Rationale makes a lot of sense to me, is a reasonable DBA perspective for permissions management